saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/content/ContentDownloadStepExecutionListener.java

```diff
@@ -0,0 +1,96 @@
+package pl.edu.icm.saos.importer.notapi.common.content;
+
+import java.io.File;
+import java.util.Collection;
+
+import org.springframework.batch.core.ExitStatus;
+import org.springframework.batch.core.StepExecution;
+import org.springframework.batch.core.StepExecutionListener;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import pl.edu.icm.saos.importer.notapi.common.ImportFileUtils;
+import pl.edu.icm.saos.persistence.model.importer.RawSourceJudgment;
+
+import com.google.common.collect.Lists;
+
+/**
+ * Listener responsible for copying content files.
+ * 
+ * @author madryk
+ */
+public class ContentDownloadStepExecutionListener implements StepExecutionListener {
+    
+    private ImportFileUtils importFileUtils;
+    
+    private ImportContentFileUtils importContentFileUtils;
+    
+    private ContentSourceFileFinder importContentFileFinder;
+    
+    private String importDir;
+    
+    private String importContentDir;
+    
+    private Class<? extends RawSourceJudgment> rawJudgmentClass;
+    
+    
+    //------------------------ LOGIC --------------------------
+    
+    @Override
+    public void beforeStep(StepExecution stepExecution) {
+        
+        importContentFileUtils.deleteAllContentFiles(rawJudgmentClass);
+        
+        File contentSourceDir = new File(importContentDir);
```

Comment:
very misleading, if you see contentSourceDir and importContentDir then you think these are different folders and then it turns out that this is the same folder: once as a string path and once as a file.